resources/lang/en/users.php

```diff
@@ -63,7 +60,8 @@
             ],
         ],
         'first_members' => 'here since the beginning',
-        'is_supporter' => 'osu!supporter',
+        'is_supporter' => 'osu!Supporter',
```

Comment:
oh right, this and one blow should be in all lowercase. The one in design image is wrong.